stdlib/public/runtime/SwiftRT-ELF.cpp

```diff
@@ -34,14 +34,32 @@ DECLARE_SWIFT_SECTION(swift5_fieldmd)
 DECLARE_SWIFT_SECTION(swift5_assocty)
 DECLARE_SWIFT_SECTION(swift5_replace)
 DECLARE_SWIFT_SECTION(swift5_replac2)
+DECLARE_SWIFT_SECTION(swift5_builtin)
+DECLARE_SWIFT_SECTION(swift5_capture)
```

Comment:
Please split this part of the change out into its own change.  If you are adding new fields to this, please bump the metadata version and ensure that we check the version before accessing the new fields.

Reply:
There's no stable ABI on Linux. Versioning doesn't seem helpful at this stage.

Reply:
Yeah, I suppose so.  I really would like to figure out how to get to a stable ABI for swift on Linux.